.github/workflows/validation.yaml

```diff
@@ -124,11 +124,16 @@ jobs:
 
       - name: Get Docker Desktop latest build url
         uses: mavrosxristoforos/get-xml-info@1.1.1
-        id: get-build-url
+        id: get-latest-build-url
         with:
           xml-file: appcast.xml
           xpath: //channel/link
 
+      - name: Define the build URL to use
+        id: get-build-url
+        run:
+          echo "info=${{ env.DD_BUILD_URL || steps.get-latest-build-url.outputs.info }}" >> $GITHUB_OUTPUT
```

Comment:
If we define the `DD_BUILD_URL` in the actions env var with a build URL such as `https://desktop.docker.com/mac/main/amd64/122432/Docker.dmg` the job downloads this specific value.